encryption-client/src/main/java/org/wso2/mi/tooling/security/EncryptionTool.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.mi.tooling.security;
+
+import org.wso2.mi.tooling.security.output.OutputManager;
+import org.wso2.mi.tooling.security.utils.IOUtils;
+
+import java.util.Map;
+import java.util.Properties;
+import javax.crypto.Cipher;
+
+public class EncryptionTool {
+
+    public static void main(String[] args) {
+
+        String keyStoreSource = System.getenv(Constants.PROP_KEYSTORE_SOURCE);
+
+        if (keyStoreSource == null) {
+            keyStoreSource = IOUtils.getDefaultLocation(Constants.PROP_KEYSTORE_DEFAULT_SOURCE);
+        }
+        Cipher cipher = EncryptionUtils.initializeCipher(IOUtils.getProperties(keyStoreSource));
+
+        String secretSource = System.getenv(Constants.PROP_SECRET_SOURCE);
+        if (secretSource == null) {
+            secretSource = IOUtils.getDefaultLocation(Constants.PROP_SECRET_DEFAULT_SOURCE);
+        }
+
+        Properties encryptionInformation = IOUtils.getProperties(secretSource);
```

Comment:
encryptionInformation == secrets, isn't it?

Reply:
encryptionInformation == secret-info.properties which has information regarding the secrets to be encrypted. i.e: output type, secrets file.